src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -32,6 +31,17 @@
 import com.revrobotics.CANSparkMax.ControlType;
 
 public class SwerveModule extends SubsystemBase {
+    public static final class SwerveModuleConstants{
+        public static final double kWheelDiameterMeters = Units.inchesToMeters(3.5);
+        public static final double kDriveMotorGearRatio = (14.0 / 50.0) * (27.0 / 17.0) * (15.0 / 45.0);
+        public static final double kTurningMotorGearRatio = (14.0 / 50.0) * (10.0 / 60.0);
+        public static final double kDriveEncoderRotFactor = kDriveMotorGearRatio * Math.PI * kWheelDiameterMeters; //Conversion factor converting the Drive Encoder's rotations to meters
+        public static final double kDriveEncoderRPMFactor = kDriveEncoderRotFactor / 60; //Conversion factor converting the Drive Encoder's RPM to meters per second
+        public static final double kTurningEncoderRotFactor = kTurningMotorGearRatio * 2 * Math.PI; //Conversion factor converting the Turn Encoder's rotations to Radians
+        public static final double kTurningEncoderRPMFactor = kTurningEncoderRotFactor / 60; //Conersion factor converting the Turn Encoder's RPM to radians per second
+        public static final double kPTurning = 0.4; //P gain for the turning motor
+    }
```

Comment:
Since we moved the constants here I think the extra SwerveModuleConstants class container is no longer needed. Same for the other constants classes except probably MotorDefaults.